### ISSUE TITLE
Add WiFi exclusion list 

### ISSUE THREAD

**Issue by Zero3K** [iTXTech/Daedalus#15]
It would be nice if an exclusion list was added that lets users exclude certain WiFi networks from having their DNS server changed from what they originally provide.

**Comment by PeratX** [comment]
Nice suggestion. Now working in progress.

**Comment by Zero3K** [comment]
Yes, I meant for it to automatically disable/use the DHCP provided DNS server when connected to certain WiFi networks.

**Comment by PeratX** [comment]
OK. How about this plan:
* Add an option in settings, like DNS Test Servers
* Add a receiver to listen when network status is changed and disable server automatically

**Comment by Zero3K** [comment]
That looks good to me.

**Comment by Zero3K** [comment]
Any progress being made on the implementing of it?

**Comment by PeratX** [comment]
Just started

**Comment by PeratX** [comment]
I found this will increase the power consume. I uses and BroadcastReceiver to handle it.

**Comment by Zero3K** [comment]
How much more does it increase it?

**Comment by PeratX** [comment]
Well, you can see a warning when using it in IntelliJ IDEA says it has been deprecated.
I haven't found a better way to do it yet.

About 5% per day. There is prototype now.

**Comment by Zero3K** [comment]
Prototype where?

**Comment by PeratX** [comment]
I will push it into a new branch later.

**Comment by PeratX** [comment]
Now I am looking into OpenVPN for Android to see whether there is a better solution.

**Comment by TeraDew** [comment]
谢谢，现在我已离校，没有了ipv6环境，不能帮忙测试，非常抱歉。

**Comment by PeratX** [comment]
不客气，现在已经完整支持IPv6。你可以试试这个：http://6plat.org/
